app/Controllers/LoginController.php

```diff
@@ -4,10 +4,19 @@
 
 namespace App\Controllers;
 
+use App\View;
+
 class LoginController
 {
-    public function login(): string
-    {
-        return 'login';
-    }
+//    public function login(): View
```

Comment:
@michael-west-aligent we should refrain from leaving commented out code in commits where possible